src/QCheck.ml

```diff
@@ -783,11 +795,17 @@ module Test = struct
     | CR_continue
     | CR_yield of 'a TestResult.t
 
+  (* Get the name of a test *)
+  let name_ cell = match cell.name with
```

Comment:
Done in commit https://github.com/c-cube/qcheck/pull/16/commits/5f5714479bc2593a1ca93d9dd1b630b8d22a8380